Engine/source/navigation/navMesh.cpp

```diff
@@ -113,7 +113,11 @@ DefineConsoleFunction(NavMeshUpdateAll, void, (S32 objid, bool remove), (0, fals
    for(U32 i = 0; i < set->size(); i++)
    {
       NavMesh *m = static_cast<NavMesh*>(set->at(i));
-      m->buildTiles(obj->getWorldBox());
+      if (m)
```

Comment:
`static_cast` doesn't return NULL the same way `dynamic_cast` does, so this check is redundant! Unless you're worried about `set->at` returning NULL?